### DISCUSSION TITLE
emergency stop

### DISCUSSION THREAD

**Discussion by JaouaherBelgacem** [bitcraze/discussions#831] [Q&A] [answered]
Hello everyone, I am new to the crazyflie and I followed the instructions. I want to use location tracking with the lighthouse. However, I have a problem when I press connect to the crazyflie the emergency stop is launched automatically and I don't understand why? I downgraded my Python to 3.8 since I had the same problem with the 3.10 and reinstalled everything again but same problem.

**Comment by knmcguire** [comment]
Hi! This is probably due to the [new supervisor triggering](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/supervisor/states/) something but can't be sure.

We need two thing from you to grasp the full picture of the firmware, OS and everything:
- The printout of the console tab when connecting to the crazyflie
- the output of the menuitem 'help'> 'about' >'debug' while connecting with the crazyflie.

**Reply by knmcguire** [reply]
Also a question based on your description of the [duplicate question](https://github.com/orgs/bitcraze/discussions/833). Does the Crazyflie emergency stop or does the CFclient 'emergency stop' (aka shutsdown). If it is the latter than provide the following information manually:
- CFclient version
- Computer OS
- Version of the Crazyflie itself

**Reply by JaouaherBelgacem** [reply]
Thank you for your reply!
well! As soon as I press on connect on the cfclient firmware, CFclient 'emergency stop' (aka shutdown) and I return back to the cmd.
here is a screen record link of what happens when I try to connect:
https://drive.google.com/file/d/1sEcGIM53tOG43hWffwEx1bZzH__FvvDJ/view?usp=sharing

The debug firmware output:

Cfclient
Cfclient version: 2023.6
System: win32
Python: 3.8.0
Qt: 5.15.2
PyQt: 5.15.9

Interface status
radio: Crazyradio version 0.53
UsbCdc: No information available
udp: None
prrt: No information available
cpx: None

Input readers
PySDL2 (0 devices connected)

Input devices
None

Crazyflie
Connected: None
Firmware: None

Decks found

Sensors found

Sensors tests

The version of the crazyflie is 2.1.

**Reply by knmcguire** [reply]
the emergency stop is not an error from the cfclient, but it's a button for the crazyflie to stop flying. But I understand the confusion

**Comment by knmcguire** [comment, marked as answer]
I've actually came across exactly your issue as well! It's a bit rare so this is the first time that I've seen this outside of my client.

You can take a look at[ this ticket,](https://github.com/bitcraze/crazyflie-clients-python/issues/656) as there are some things that can prevent it from crashing... this means that you'll need to install the cfclient from source and adjust the source code. You can find those instructions here: https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/#windows-781011

This is an annoying problem and we haven't found a way to fix it yet as we were not able to reproduce this on someone else's windows.

**Reply by JaouaherBelgacem** [reply]
Thanks!

**Reply by JaouaherBelgacem** [reply]
Hi!
I commented the lines that you did from 76 to 81 and unfortunately I still have the same problem 

note: I have windows 10.

Could python version 3.8 is the problem?

**Reply by JaouaherBelgacem** [reply]
I finally worked. I commeted the metioned lines out and restarted my laptop and then it worked
